apps/client/src/components/Conclusion/ConclusionOutcome.tsx

```diff
@@ -35,51 +30,54 @@ export const ConclusionOutcome: React.FC<Props> = ({
     window.print();
   };
 
-  const handlePermitButton = () => {
-    matomoTrackEvent({
-      action: actions.CLICK_EXTERNAL_NAVIGATION,
-      name: eventNames.APPLY_FOR_PERMIT,
-    });
-    // Open OLO in new tab/window
-    window.open(Olo.home, "_blank");
+  const conclusionText = {
+    needPermit: {
+      title: "U hebt een omgevingsvergunning nodig.",
+      contentParagraph: (
+        <NeedPermitContent matomoTrackEvent={matomoTrackEvent} />
+      ),
+      finalParagraph: <NeedPermitFooter />,
+    },
+    needContact: {
+      title: contactConclusion?.title,
```

Comment:
The title is required so you shouldn't need a questionmark. Is TS complaining?

Reply:
Yes, this is because of typescript